lib/analyze/util/normalizePackageJson.js

```diff
@@ -7,39 +7,19 @@ const log = logger.child({ module: 'util/normalize-package-json' });
 /**
  * Normalizes a package.json.
  *
- * @param {string} name        The package name
+ * Wrapper around normalize-package-data module that handles extra stuff, see code below.
+ *
  * @param {object} packageJson The package package.json
- * @param {object} [options]   The options; read below to get to know each available option
  *
  * @return {object} The normalized package.json
  */
-function normalizePackageJson(name, packageJson, options) {
-    options = Object.assign({ checkName: true }, options);
-
-    // Check if there's no name (e.g.: 6to5-runtime)
-    if (options.checkName) {
```

Comment:
They no longer add any value. All validation is made in `https://github.com/npms-io/npms-analyzer/blob/monorepos/lib/analyze/util/packageJsonFromData.js#L26` and since the `preferDownloaded` no longer exists, it's not necessary at all.